doc/config.yml

```
@@ -33,11 +35,24 @@ Extensions:
         name: FENIX
         repo: https://github.com/idaholab/fenix
         menu:
-            Documentation and Resources:
-                Complete Code Manual: syntax/index.md
-                Code Standards: sqa/fenix_scs.md
-                Contributing to FENIX: /sqa/contributing.md
+            Getting Started:
+                Installation: getting_started/installation.md
+                FENIX on HPC: getting_started/hpc.md
+                Using FENIX: getting_started/using_fenix.md
+                User Notes: getting_started/user_notes.md
+                Contributing to FENIX: sqa/contributing.md
+            Documentation:
+                FENIX-only Syntax: syntax/fenix_only.md
+                Complete Code Syntax: syntax/index.md
+                V&V and Examples: /verification_validation_examples/index.md
+                Publications using FENIX: publications.md
+            Software Quality:
                 Software Quality Assurance (SQA) Information: /sqa/index.md
+                Code Standards: sqa/fenix_scs.md
```

```suggestion
                Code Standards: sqa/fenix_scs.md
                Code Coverage: https://mooseframework.inl.gov/fenix/docs/coverage
```

This URL isn't live yet, but we generate coverage on every merge to main. CIVET supposedly publishes the results here (see https://civet.inl.gov/job/2210582/). Likely just need to make the spot for it on the server public or re-route it to `https://fenix-dev.hpc.inl.gov/docs/coverage`. @loganharbour can you advise and assist with the best route forward for now? FENIX should be open-source on the order of months.

---

I added this suggestion, but leaving this discussion as unresolved since @loganharbour needs to jump in.

---

Will revisit as part of addressing #35. Resolving.